SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -331,5 +341,19 @@ private static string Quote(string toQuote)
         {
             return $"\"{toQuote}\"";
         }
+
+        private static void AppendProperyOverrideText(string workerPath, string gameName)
```

Comment:
`AppendProperyOverrideText`? Maybe something more descriptive like `ForceSpatialNetworkingInConfig`

Reply:
When writing this I figured we might want to also override other properties in the future. But I agree that naming can always be changed, and that a more descriptive name is appropriate here. Fixed it now.